jbpm-case-mgmt/jbpm-case-mgmt-impl/src/main/resources/META-INF/CaseMgmtorm.xml

```diff
@@ -14,8 +14,8 @@
         log.processInstanceDescription, <!-- case description -->            
         log.processId, <!--case definition  -->
         log.status, <!-- case status -->
-        log.externalId, <!-- deployment id -->      
-        log.identity, <!-- case owner -->
+        log.externalId, <!-- deployment id -->
+        (select r.entityId from CaseRoleAssignmentLog r where r.processInstanceId = log.processInstanceId and r.roleName = 'owner' and r.type = 1), <!-- case owner -->
```

Comment:
I believe this will only work for case definitions that have owner defined or any other role defined as then the owner is added automatically. But if case definition is not using roles at all that will be then empty, isn't it?